### PR TITLE
[FE-16840] Change cross section config to support new vega json format

### PR DESCRIPTION
**Pull request by cmatzenbach**
Changes vega json rendered by Cross Section layer to match new renderer API.

### CODE REVIEW

File: src/mixins/render-vega-lite/Definitions/Transform/Transforms/CrossSection2dDefinitionObject.js [hunk 1]
```diff
@@ -183,11 +246,12 @@ export default class CrossSection2dDefinitionObject extends PropertiesDefinition
         type: "cross_section2d",
         coords: {
           x: this.x,
-          y: this.y,
-          z: this.z
+          y: this.y
         },
         xy_cross_section: this.crossSectionLine,
-        cross_section_dimension: this.crossSectionDimensionName
+        num_points_x: this.numPointsX,
```
Comment (johallar) [inline src/mixins/render-vega-lite/Definitions/Transform/Transforms/CrossSection2dDefinitionObject.js:252]: Do these need to be camel case now? Thought @simoneves had made that change on the BE side

Comment (cmatzenbach) [reply src/mixins/render-vega-lite/Definitions/Transform/Transforms/CrossSection2dDefinitionObject.js]: Fixed - @johallar ready for re-review

**Review by johallar — APPROVED** [review]
LGTM!

**Comment by cmatzenbach** [comment]
> @cmatzenbach Can you build and push the `dist` files?

Gah good call, why do I _always_ forget that....

**Comment by johallar** [comment]
> > @cmatzenbach Can you build and push the `dist` files?
> 
> Gah good call, why do I _always_ forget that....

We need to get to the bottom of why those are included (and remove those bad boys), I think when its installed as a dependency it gets cloned and built...

**Pull request merged**